osmdroid/src/main/java/org/odk/collect/osmdroid/OsmDroidMapFragment.java

```diff
@@ -863,6 +895,68 @@ public void removeLastPoint() {
         }
     }
 
+    private class PolygonFeature implements MapFeature {
```

Comment:
It would be better to have it in a separate class as we do in mapbox but that would require factoring out MarkerFeature and PolyLineFeature too so probably a thing for the future.